Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Mail/base_mail.yml

```diff
@@ -93,6 +95,16 @@
     damage:
       types:
         Blunt: 10
+  - type: CargoSellBlacklist # Frontier
+  - type: Food # Frontier - Moth food
```

Comment:
Why can moths eat paper?

Reply:
🤷 I feel like this was nuked on frontier as a thing later because technically that gives moths an infinite food source... Why can they eat paper? Some moths eat paper, apparently.